### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by colinkiama** [nahuelwexd/vala-website#10] [General]
@Suzie97

**Comment by Suzie97** [comment]
Initial text styles and the colour palette (which is straight copy of the elementary palette for now) have been created in the Figma file.

**Reply by pervoj** [reply]
Where can we reach the mockups in Figma?

**Reply by colinkiama** [reply]
https://www.figma.com/file/hOC3cJpOmUmWjKUdkXpmBH/Home-Page?node-id=0%3A1

**Comment by Suzie97** [comment]
As for the Information Architecture, I think this needs some research so that we are clear about what our audience, their goals, needs and wants. 

We can setup a form which can be sent to potential users, so that we have some actual statistical data about what our users want, and then continue to think about the content of the website based on the results.

Let me know if this is feasible, and worth it in your opinion.

**Reply by colinkiama** [reply]
I think that this is feasible however, I don’t think that should be done immediately.

I think that the top priorities for the initial launch should be:
- Giving the language a home that is outside of the GNOME Wiki.
- Making the site easy to maintain
- Making the Vala repository and communities easily disoverable
    - Visitors should be able to access the source from any page on the site in one click 

The existing design has been approved by the community on Matrix already and we have guidance from existing issues in this repo like: https://github.com/colinkiama/vala-www/issues/1

**Comment by Suzie97** [comment]
<img width="953" alt="Screenshot 2022-05-05 at 01 02 32" src="https://user-images.githubusercontent.com/68198116/166860084-1dce5161-c851-4e13-9c9b-e64f34e61db9.png">

Initial concept for the Who Uses Vala section. The idea is to show off the more important projects like this and mention other ones using text only below them.

**Reply by Suzie97** [reply]
@colinkiama was talking about having to big click area so people can just click on these "cards" and check them out.

**Reply by nyx-lyb3ra** [reply]
I'm loving this 🥰

Colin's idea sounds good to me… but I can't think how it would work with GNOME :thinking: IIRC GitLab doesn't have a way to filter projects by language, unless we use the GitHub mirror.

**Reply by dar5hak** [reply]
@Suzie97
- elementary OS Community: Drop "really", "third party" and "major ones like". It's terser and would match the number of lines to its left counterpart.
- Birdfont: "which _is_ made in Vala"
- Pamac:
  - "that focuses on providing" → "that provides" (or replace it with simply a comma)
  - "easy to use" → "easy-to-use" (or just "easy")

**Reply by Suzie97** [reply]
@nahuelwexd any ideas for what to write for GNOME? And is there any distinction between official gnome and community like the different logos in the elementary community?

**Reply by pervoj** [reply]
@Suzie97 there is GNOME Circle, but it doesn't include all GNOME community projects

**Reply by Suzie97** [reply]
> @Suzie97 there is GNOME Circle, but it doesn't include all GNOME community projects

It's not all vala exclusive apps

**Reply by Suzie97** [reply]
Here is an idea, we can use awesomeopensource.com and then filter searches to show Vala and GNOME apps only, then straight up link to that

https://awesomeopensource.com/projects/gnome/vala

**Reply by nyx-lyb3ra** [reply]
@Suzie97 There is no official logo for third-party projects in GNOME. There is what is called the GNOME Circle, but it does not include all projects, as you need to meet a number of requirements and apply to be part of it.

**Reply by lw64** [reply]
I would just pick 10 good-looking and still maintained apps and copy the icons over into this repo.

**Comment by colinkiama** [comment]
How do you all feel about the latest  long-term support and stable and version numbers and release dates being on the homepage somewhere?

**Reply by pervoj** [reply]
I think it's a good idea

**Reply by lw64** [reply]
That is only really useful for downstream developers and for a few other cases I think, and shouldn't be on the main page.

**Reply by dar5hak** [reply]
That probably belongs on an installation guide.

**Comment by lw64** [comment]
I have a few issues with the current design:
1. The violet and white have a really hard contrast and also something like a secondary color or a bit more soft/relaxed looking would be good.
2. The code view in the middle takes extremely much space. I think on Desktop it could go to the right side, so on the left could go text.

**Reply by lw64** [reply]
Here is a quick draft how I could imagine the website:
[vala-website.pdf](https://github.com/colinkiama/vala-www/files/8636228/vala-website.pdf)

**Reply by colinkiama** [reply]
I like the layout, it reminds me of the old design: https://github.com/colinkiama/vala-www/discussions/37
